BrowserSelect/Program.cs

```diff
@@ -20,6 +20,8 @@ static class Program
         [STAThread]
         static void Main(string[] args)
         {
+
+            System.Diagnostics.Debug.WriteLine("Test");
```

Comment:
?

Reply:
Sorry, I was statically printing out the JSON being generated to debug the Icon->String->PNG converter, you can remove that line.